project.yml

```diff
@@ -0,0 +1,143 @@
+name: Kiwix
+options:
+  xcodeVersion: 13.0
+  deploymentTarget:
+      iOS: 15.0
+      macOS: 12.0
+  generateEmptyDirectories: true
+  useTabs: false
+  groupSortPosition: top
+
+settings:
+  base:
+    DEVELOPMENT_TEAM: L7HWM3SP3L
```

Comment:
This should not be a secret?

Reply:
It is not considered a secret, please see: https://stackoverflow.com/questions/44417171/should-i-keep-the-apple-developer-team-id-secret

Reply:
We were also including this value in the original pbxproject file (several times actually).